lib/hal_api/controller/actions.rb

```diff
@@ -9,7 +9,7 @@ def show
 
   def create
     create_resource.tap do |res|
-      consume_with_content_type! res
+      consume_with_content_type! res, create_options
```

Comment:
looks like we could just be using `consume!` now, this method is now fixed in roar-rails
https://github.com/apotonick/roar-rails/blob/v1.0.1/lib/roar/rails/controller_additions.rb#L28